sphinx/source/index.rst

```diff
@@ -33,22 +33,23 @@ the :ref:`devguide`.
     docs/user_guide/setup
     docs/user_guide/concepts
     docs/user_guide/plotting
-    docs/user_guide/categorical
-    docs/user_guide/graph
-    docs/user_guide/geo
-    docs/user_guide/compat
-    docs/user_guide/annotations
     docs/user_guide/styling
+    docs/user_guide/annotations
     docs/user_guide/tools
     docs/user_guide/layout
     docs/user_guide/notebook
-    docs/user_guide/interaction
-    docs/user_guide/cli
     docs/user_guide/server
-    docs/user_guide/embed
+    docs/user_guide/interaction
+    docs/user_guide/data
+    docs/user_guide/categorical
+    docs/user_guide/graph
+    docs/user_guide/geo
```

Comment:
I think there's some conflict with some other re-arrangements. I'd like the following order;

docs/user_guide/setup
docs/user_guide/concepts
docs/user_guide/plotting
docs/user_guide/data
docs/user_guide/layout
docs/user_guide/categorical
docs/user_guide/graph
docs/user_guide/geo
docs/user_guide/tools
docs/user_guide/styling
docs/user_guide/annotations
docs/user_guide/interaction
docs/user_guide/server
docs/user_guide/notebook
docs/user_guide/export
docs/user_guide/embed
docs/user_guide/extensions
docs/user_guide/compat
docs/user_guide/webgl
docs/user_guide/bokehjs
docs/user_guide/tutorials
docs/user_guide/info